R/binning.R

```diff
@@ -108,3 +108,47 @@ bin <- function(x, binmode = c("equalN", "equalWidth", "breaks"),
     res
 }
 
+#' Get and set the zero bin manually
+#' 
+#' @param bins Factor, typically the return value of \code{\link[monaLisa]{bin}}.
+#' @param zeroBin Numeric or character scalar indicating the level to use as 
+#'   the zero bin.
+#' 
+#' @examples 
+#' set.seed(1)
+#' x <- rnorm(100)
+#' bins <- bin(x, "equalN", nElements = 10, minAbsX = 0.5)
+#' getZeroBin(bins)
+#' bins <- setZeroBin(bins, 2)
+#' 
+#' @return 
+#' For \code{getZeroBin}, the index of the level representing the zero bin. 
+#' For \code{setZeroBin}, a modified factor with the zero bin set to the 
+#' provided value.
+#' 
+#' @name getSetZeroBin
+NULL
+
+#' @export
+#' @rdname getSetZeroBin
+getZeroBin <- function(bins) {
+    attr(bins, "bin0")
+}
+
+#' @export 
+#' @rdname getSetZeroBin
+setZeroBin <- function(bins, zeroBin) {
```

Comment:
I have replaced `stopifnot()` by `.assertScalar` and `.assertVector` - we have them in `monaLisa` anyway and I think they give more informative error messages. I hope you are ok with that change.

Reply:
Oh, yes - great!